sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/eventsourcedentity/EventSourcedEntity.java

```diff
@@ -116,6 +135,23 @@ protected final Effect.Builder<S, E> effects() {
   }
 
   /**
+   * An Effect is a description of what Kalix needs to do after the command is handled.
+   * You can think of it as a set of instructions you are passing to Kalix. Kalix will process the instructions on your
+   * behalf and ensure that any data that needs to be persisted will be persisted.
+   * <p>
+   * Each Kalix component defines its own effects, which are a set of predefined
+   * operations that match the capabilities of that component.
+   * <p>
+   * An EventSourcedEntity Effect can either:
+   * <p>
+   * <ul>
+   *   <li>emit events and send a reply to the caller
+   *   <li>ignore the command by simply returning to the caller without emitting any events
```

Comment:
There is not `ignore` in this class.

Reply:
No, maybe my sentence is not clear enough. I will remove the 'ignore' to avoid confusion. I mean to say, you can handle a command but decide to not mutate the state and simply return an acknowledgement.

Reply:
I removed this sentence. Confusing and the use case that I was trying to explain is something that the user can easily learn by themselves.